.kitchen.yml

```diff
@@ -41,7 +46,7 @@ provisioner:
       source: https://github.com/salt-formulas/salt-formula-linux.git
     - name: git
       repo: git
-      source: https://github.com/salt-formulas/salt-formula-git.git
+      source: https://github.com/gtmanfred/salt-formula-git.git
```

Comment:
This seems ... odd? Did we pull from upstream master? did we maybe want to pull from an upstream tag?

Reply:
I pulled from the v0.6.0 tag from git@github.com:saltstack/kitchen-salt.git